CHANGELOG.md

```diff
@@ -36,6 +37,7 @@
 - Added support for modified gravity via mu / Sigma (scale-independent) parameterisation (#442)
 
 ## C library
+- Added function for comoving angular diameter distance (#645)
```

Comment:
Neither of these changelog entries are accurate. We are computing the angular diameter distance difference.

Reply:
Done